firebase-functions/src/main/java/com/google/firebase/functions/HttpsCallableReference.java

```diff
@@ -89,6 +94,28 @@
    * @return A Task that will be completed when the HTTPS request has completed.
    */
   public Task<HttpsCallableResult> call() {
-    return functionsClient.call(name, null);
+    return functionsClient.call(name, null, options);
+  }
+
+  /**
+   * Changes the timeout for calls from this instance of Functions. The default is 60 seconds.
+   *
+   * @param timeout The length of the timeout, in the given units.
+   * @param units The units for the specified timeout.
+   */
+  public void setTimeout(long timeout, TimeUnit units) {
```

Comment:
Does this require an API review?

Reply:
Yes, the API has been approved. go/callable-timeouts